packages/scripts/README.md

```diff
@@ -435,13 +436,38 @@ This is how you execute those scripts using the presented setup:
 * `npm run test:unit` - runs all unit tests.
 * `npm run test:unit:help` - prints all available options to configure unit tests runner.
 * `npm run test:unit:watch` - runs all unit tests in the watch mode.
+* `npm run test:unit:debug` - runs all unit tests with the node debugger enabled.
 
 Jest will look for test files with any of the following popular naming conventions:
 
 - Files with `.js` (or `.ts`) suffix located at any level of depth in `__tests__` folders.
 - Files with `.js` (or `.ts`) suffix directly located in `test` folders.
 - Files with `.test.js` (or `.test.ts`) suffix.
 
+#### Debugging Jest unit tests
+
+Tests can be debugged in Google Chrome, Visual Studio Code, or any other [Inspector Client](https://nodejs.org/en/docs/guides/debugging-getting-started/#inspector-clients).
+
+##### Debugging in Google Chrome
+
+Place `debugger;` statements in any test and run `npm run test:unit:debug`.
+
+Then open `about:inspect` in Google Chrome and select `inspect` on your process.
```

Comment:
What about other browsers? Would it also work with Firefox or Safari?

Reply:
Thanks for clarification 👍 The only thing that I feel should be updated is to make it more generic like the section should refer to debugging in the browser. It would be helpful to make it also clear that Chrome isn’t the only browser option but we use it to better illustrate the workflow. It’s mostly cosmetic changes to what you included.